src/SDK/Subscriber/SubscribedEvent.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Subscriber;
+
+use function count;
+use OpenTelemetry\SDK\Attributes;
+use OpenTelemetry\SDK\AttributesInterface;
+use OpenTelemetry\SDK\Trace;
+use OpenTelemetry\SDK\Trace\Span;
+
+final class SubscribedEvent implements SubscribedEventInterface
+{
+	
+	private Span $span;
+
+	public function __construct(Span $span)
+	{
+		$this->span = $span;
+	}
+
+	public function dispatch():void
+	{
+		$classname = get_called_class();
+		$classname = str_replace('OpenTelemetry\\SDK\\Subscriber\\','',$classname);
+		$listenerClassName = $this->span->getListener($classname);
+		$listenerClassName = "OpenTelemetry\\SDK\\Subscriber\\Listener\\".$listenerClassName;
```

Comment:
This looks unreliable.
How do you want to test this?

Reply:
In the implementation the process the user could use to register the Listener with Event is as commented above. Now to register the Listener class the class file should be present in a fixed folder that's why I am using a static path.